core/embed/prodtest/version.h

```diff
@@ -1,6 +1,6 @@
 #define VERSION_MAJOR 0
 #define VERSION_MINOR 2
-#define VERSION_PATCH 2
+#define VERSION_PATCH 4
```

Comment:
Why skip 3?

Reply:
it's there in the commits actually :) first go to 0.2.3 for a commit that will be tagged, and bump immediately after in "bump version after release"